src/components/LazyLoadImage.tsx

```diff
@@ -0,0 +1,60 @@
+import Image from 'next/image';
+import React, { useState, useRef, useEffect } from 'react';
+
+const registerObserver = (
```

Comment:
what is this complex logic? NextJs Image its already loaded like that...
what i was saying is until the image isnt loaded show the skeleton

Reply:
Using this LazyLoadImage component, we can customize our own skeleton, and styling effect.